assets/js/post_office_search.tsx

```diff
@@ -11,15 +11,17 @@ import { UsStatesTerritories } from './form_helper';
 export function render() {
   const elem = document.getElementById('post-office-search')!;
   const root = createRoot(elem);
-  const { locationsSearchUrl } = elem.dataset;
+  const { locationsSearchUrl, pageLang } = elem.dataset;
```

Comment:
Could we use `document.documentElement.lang` instead of passing a new `data-` attribute?

Reply:
It seems like that default isn't ever used, even in English, so probably safe to swap it out.

Reply:
yep, changed